cli_reference/odo-important-update.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/odo-important-update.adoc
+
+:_content-type: CONCEPT
+[id="odo-important_update_{context}"]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+include::_attributes/common-attributes.adoc[]
+= Important update on `{odo-title}`
+:context: odo-important-update
+
+toc::[]
+
+Red Hat does not provide information about `{odo-title}` on the {OCP} documentation site. You can refer the link:https://odo.dev[document] maintained by the upstream community for any information related to `{odo-title}`.
+
+[IMPORTANT]
+====
+For the documents maintained by the upstream community, Red Hat provides limited support under link:https://access.redhat.com/solutions/5893251[Cooperative Community Support]. 
```

Comment:
```suggestion
For the documents maintained by the upstream community, Red Hat provides support under link:https://access.redhat.com/solutions/5893251[Cooperative Community Support]. 
```

I would remove the word limited as we still want customers to raise support tickets and support team can guide to respective team. This is to make sure when we reach out to customers, we convey that RH is fully invested in this project.

Reply:
Updated.